sdk/notificationhubs/Azure.ResourceManager.NotificationHubs/src/Generated/Models/NotificationHubNameAvailabilityResult.cs

```diff
@@ -12,15 +12,15 @@
 namespace Azure.ResourceManager.NotificationHubs.Models
 {
     /// <summary> Description of a CheckAvailability resource. </summary>
-    public partial class CheckAvailabilityResult : TrackedResourceData
+    public partial class NotificationHubNameAvailabilityResult : TrackedResourceData
     {
-        /// <summary> Initializes a new instance of CheckAvailabilityResult. </summary>
+        /// <summary> Initializes a new instance of NotificationHubNameAvailabilityResult. </summary>
         /// <param name="location"> The location. </param>
-        public CheckAvailabilityResult(AzureLocation location) : base(location)
+        public NotificationHubNameAvailabilityResult(AzureLocation location) : base(location)
```

Comment:
Similar question as NotificationHubNameAvailabilityContent, why does this model inherit from TrackedResourceData and contain a sku property?

Reply:
As above, renamed it to `NotificationHubAvailabilityResult`

Reply:
Also renamed the corresponding methods